src/specialOffer/SpecialOffer.tsx

```diff
@@ -0,0 +1,78 @@
+import React from 'react'
+
+import { Button, ButtonStatus } from '../button'
+import { Disclaimer } from '../disclaimer'
+import { Paragraph } from '../paragraph'
+import { SubHeader } from '../subHeader'
+import {
+  StyledSpecialOffer,
+  StyledSpecialOfferButton,
+  StyledSpecialOfferButtons,
+  StyledSpecialOfferContent,
+  StyledSpecialOfferFigure,
+  StyledSpecialOfferText,
+} from './SpecialOffer.style'
+
+type Picture = Readonly<{
+  url: string
+  alt: string
+}>
+
+type Link = Readonly<{
+  label: string
+  url: string
+}>
+
+type Button = Readonly<{
+  label: string
+  onClick: () => void
+}>
+
+type CTALinks = Array<Link>
+type CTAButtons = Array<Button>
+
+export type SpecialOfferProps = Readonly<{
+  picture: Picture
+  title: string
+  description: string
+  informationLink?: Link
+  actions?: CTALinks | CTAButtons
+}>
+
+export const SpecialOffer = (props: SpecialOfferProps) => {
+  const { picture, title, description, informationLink, actions = [] } = props
+
+  return (
+    <StyledSpecialOffer>
+      <StyledSpecialOfferContent>
+        <StyledSpecialOfferFigure>
+          <img src={picture.url} alt={picture.alt} />
+        </StyledSpecialOfferFigure>
+      </StyledSpecialOfferContent>
+      <StyledSpecialOfferContent>
+        <StyledSpecialOfferText>
+          <SubHeader>{title}</SubHeader>
+          <Paragraph>{description}</Paragraph>
+          {informationLink && (
+            <Disclaimer useInfoIcon>
+              <Button status={ButtonStatus.UNSTYLED} href={informationLink.url}>
+                {informationLink.label}
+              </Button>
+            </Disclaimer>
+          )}
+        </StyledSpecialOfferText>
+        <StyledSpecialOfferButtons>
+          {(actions as Array<Link | Button>).map(action => (
```

Comment:
nit: Declare this function outside render

Reply:
Mmh I don't want to declare a function `renderButton :: action -> JSX.Element` outside of `SpecialOffer` because it would not be a real component (no props signature). Also the performance impact of having an arrow function in `Array.map` should be very small (JS runtime have usually no issue creating lot of functions and the array is always very small).